testing/main_test.go

```diff
@@ -368,62 +403,62 @@ FROM
+	}
+}
+
+func nothingSkipped() map[authMode]string { return nil }
+
 func TestGORM(t *testing.T) {
-	testORM(t, testInfo{language: "go", orm: "gorm"})
+	testORMForAuthModesExcept(t, testInfo{language: "go", orm: "gorm"}, nothingSkipped())
 }
 
 func TestGOPG(t *testing.T) {
-	testORM(t, testInfo{
-		language: "go",
-		orm:      "gopg",
-		// GoPG does not support client certs:
-		// https://github.com/go-pg/pg/blob/v10/options.go
-		// If we set up a secure deployment and went through the proxy, it would work (or should anyway), but only
-		// via the 'database' parameter; GoPG also does not support the 'options' parameter.
-		insecure: true,
-	})
+	testORMForAuthModesExcept(t,
+		testInfo{language: "go", orm: "gopg"},
+		map[authMode]string{
+			// https://github.com/go-pg/pg/blob/v10/options.go
+			// If we set up a secure deployment and went through the proxy, it would work (or should anyway), but only
+			// via the 'database' parameter; GoPG also does not support the 'options' parameter.
+			//
+			// pg: options other than 'sslmode', 'application_name' and 'connect_timeout' are not supported
+			authClientCert: "GoPG does not support custom root cert",
+			authPassword:   "GoPG does not support custom root cert",
+		})
 }
 
 func TestHibernate(t *testing.T) {
-	testORM(t, testInfo{
-		language: "java",
-		orm:      "hibernate",
-		// Possibly does not unescape the path correctly:
-		// Caused by: java.io.FileNotFoundException:
-		//	%2Ftmp%2Fcockroach-testserver913095208%2Fcerts%2Fca.crt (No such file or directory)
-		insecure: true,
-	})
+	testORMForAuthModesExcept(t, testInfo{language: "java", orm: "hibernate"}, nothingSkipped())
 }
 
 func TestSequelize(t *testing.T) {
-	testORM(t, testInfo{
-		language: "node",
-		orm:      "sequelize",
-		// Requires bespoke code to actually use SSL, see:
-		// https://github.com/sequelize/sequelize/issues/10015
-		insecure: true,
-	})
+	testORMForAuthModesExcept(t, testInfo{language: "node", orm: "sequelize"}, nothingSkipped())
 }
 
 func TestSQLAlchemy(t *testing.T) {
-	testORM(t, testInfo{
-		language: "python",
-		orm:      "sqlalchemy",
-	})
+	testORMForAuthModesExcept(t, testInfo{language: "python", orm: "sqlalchemy"}, nothingSkipped())
 }
 
 func TestDjango(t *testing.T) {
-	testORM(t, testInfo{
-		language:    "python",
-		orm:         "django",
-		tableNames:  djangoTestTableNames,
-		columnNames: djangoTestColumnNames,
-		// No support for client certs (at least not via the query string).
-		// psycopg2.OperationalError: fe_sendauth: no password supplied
-		insecure: true,
-	})
+	testORMForAuthModesExcept(
+		t,
+		testInfo{
+			language:    "python",
+			orm:         "django",
+			tableNames:  djangoTestTableNames,
+			columnNames: djangoTestColumnNames,
+		}, nothingSkipped(),
+	)
 }
 
 func TestActiveRecord(t *testing.T) {
-	testORM(t, testInfo{language: "ruby", orm: "activerecord"})
+	testORMForAuthModesExcept(t, testInfo{language: "ruby", orm: "activerecord"}, nothingSkipped())
 }
+
```

Comment:
is this extra \n from gofmt?
wonder if we even have a linter

Reply:
it was accidental -- no there's no linter on this project :/